qiita_pet/templates/sitebase.html

```diff
@@ -570,6 +570,33 @@
         </div>
       </div> <!--/.main nav-bar -->
 
+      <div class="modal fade qiita_pet_dwnld_confirm" tabindex="-1" role="dialog" aria-labelledby="Confirm download">
+        <div class="modal-dialog modal-lg" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title">File really big.</h4>
+            </div>
+            <div class="modal-body">
+              <p>The file you're trying to download is really big. Would you like to proceed?</p>
+            </div>
+            <div class="modal-footer">
+              <button type="button" class="btn btn-danger" data-dismiss="modal">No</button>
+              <button type="button" class="btn btn-success" id="qiita_dwnld_accept">Yes</button>
+            </div>
+            <!-- href="{% raw qiita_config.portal_dir %}/release/download/public" -->
+          </div>
+        </div>
+      </div>
+
+      <script type="text/javascript">
```

Comment:
Can this piece of javascript be moved up with the rest of the JavaScript in this page? Thanks!